README.md

```diff
@@ -1,2 +1,60 @@
-# snap-opensearch
-OpenSearch Snap
+# OpenSearch-Snap
+
+<h1 align="center">
+  <img src="https://opensearch.org/assets/brand/PNG/Logo/opensearch_logo_default.png" alt="OpenSearch" />
+  <br />
+</h1>
```

Comment:
How permissive is the Opensearch branding? Can we use their logo?

Reply:
Commented it for now - until we get an answer